src/components/ADempiere/Form/ProductInfo/productList.vue

```diff
@@ -83,15 +120,24 @@
         width="100"
       >
         <template slot-scope="scope">
-          {{ formatQuantity(scope.row.quantityAvailable) }}
+          {{
+            formatQuantity({
+              value: scope.row.quantityAvailable
+            })
+          }}
         </template>
       </el-table-column>
       <el-table-column
         :label="$t('form.productInfo.price')"
         align="right"
       >
         <template slot-scope="scope">
-          {{ formatPrice(scope.row.priceStandard, scope.row.currency.iSOCode) }}
+          {{
```

Comment:
Just can see change for soiurce code with the same functionality, please let it without changes